### PR TITLE
Fixes #2784

### LINKED ISSUE #2784: Element symbol lookup for some transuranics returns incorrect results
**Configuration:**

- RDKit Version: 2019.09.1.0
- Operating system: Ubuntu 19.04
- Python version (if relevant): 3.7.5
- Are you using conda? Yes
- If you are using conda, which channel did you install the rdkit from? rdkit
- If you are not using conda: how did you install the RDKit? N/A

**Description:**
Element symbol lookup via PeriodicTable.GetElementSymbol returns incorrect results.

Elements with atomic number > 114 are incorrectly labeled. There are additional entries above 118. It appears that some of the old symbols were not removed when the new symbols were added.

```
from rdkit.Chem import PeriodicTable
from rdkit.Chem import GetPeriodicTable
expected_atomic_nums_and_symbols = [
    (1, 'H'),
    (2, 'He'),
    (3, 'Li'),
    (4, 'Be'),
    (5, 'B'),
    (6, 'C'),
    (7, 'N'),
    (8, 'O'),
    (9, 'F'),
    (10, 'Ne'),
    (11, 'Na'),
    (12, 'Mg'),
    (13, 'Al'),
    (14, 'Si'),
    (15, 'P'),
    (16, 'S'),
    (17, 'Cl'),
    (18, 'Ar'),
    (19, 'K'),
    (20, 'Ca'),
    (21, 'Sc'),
    (22, 'Ti'),
    (23, 'V'),
    (24, 'Cr'),
    (25, 'Mn'),
    (26, 'Fe'),
    (27, 'Co'),
    (28, 'Ni'),
    (29, 'Cu'),
    (30, 'Zn'),
    (31, 'Ga'),
    (32, 'Ge'),
    (33, 'As'),
    (34, 'Se'),
    (35, 'Br'),
    (36, 'Kr'),
    (37, 'Rb'),
    (38, 'Sr'),
    (39, 'Y'),
    (40, 'Zr'),
    (41, 'Nb'),
    (42, 'Mo'),
    (43, 'Tc'),
    (44, 'Ru'),
    (45, 'Rh'),
    (46, 'Pd'),
    (47, 'Ag'),
    (48, 'Cd'),
    (49, 'In'),
    (50, 'Sn'),
    (51, 'Sb'),
    (52, 'Te'),
    (53, 'I'),
    (54, 'Xe'),
    (55, 'Cs'),
    (56, 'Ba'),
    (57, 'La'),
    (58, 'Ce'),
    (59, 'Pr'),
    (60, 'Nd'),
    (61, 'Pm'),
    (62, 'Sm'),
    (63, 'Eu'),
    (64, 'Gd'),
    (65, 'Tb'),
    (66, 'Dy'),
    (67, 'Ho'),
    (68, 'Er'),
    (69, 'Tm'),
    (70, 'Yb'),
    (71, 'Lu'),
    (72, 'Hf'),
    (73, 'Ta'),
    (74, 'W'),
    (75, 'Re'),
    (76, 'Os'),
    (77, 'Ir'),
    (78, 'Pt'),
    (79, 'Au'),
    (80, 'Hg'),
    (81, 'Tl'),
    (82, 'Pb'),
    (83, 'Bi'),
    (84, 'Po'),
    (85, 'At'),
    (86, 'Rn'),
    (87, 'Fr'),
    (88, 'Ra'),
    (89, 'Ac'),
    (90, 'Th'),
    (91, 'Pa'),
    (92, 'U'),
    (93, 'Np'),
    (94, 'Pu'),
    (95, 'Am'),
    (96, 'Cm'),
    (97, 'Bk'),
    (98, 'Cf'),
    (99, 'Es'),
    (100, 'Fm'),
    (101, 'Md'),
    (102, 'No'),
    (103, 'Lr'),
    (104, 'Rf'),
    (105, 'Db'),
    (106, 'Sg'),
    (107, 'Bh'),
    (108, 'Hs'),
    (109, 'Mt'),
    (110, 'Ds'),
    (111, 'Rg'),
    (112, 'Cn'),
    (113, 'Nh'),
    (114, 'Fl'),
    (115, 'Mc'),
    (116, 'Lv'),
    (117, 'Ts'),
    (118, 'Og')
]
# Iterate to 120 to get additional 2 entries.
rdkit_element_symbols = ([PeriodicTable.GetElementSymbol(periodic_table, atomic_num) for atomic_num in range(1, 121)])
print(rdkit_element_symbols)
for atomic_number, element_symbol in expected_atomic_nums_and_symbols:
    assert rdkit_element_symbols[atomic_number - 1] == element_symbol
```

### PR DESCRIPTION
**Pull request by greglandrum**
Small change without breaking backwards compatibility

### CODE REVIEW

**Comment by greglandrum** [comment]
@d-b-w : this would be a pretty crappy (and fragile) way of adding aliases.
I agree that would be a useful thing to be able to do, but I would rather add a new feature for that instead of (ab)using the current `PeriodicTable` API.

File: Code/GraphMol/PeriodicTable.cpp [hunk 1]
```diff
@@ -35,9 +35,16 @@ PeriodicTable::PeriodicTable() {
        ++token) {
     if (*token != " ") {
       atomicData adata(*token);
-      byanum.push_back(adata);
       std::string enam = adata.Symbol();
       byname[enam] = adata.AtomicNum();
+      // there are, for backwards compatibility reasons, some duplicate rows for
+      // atomic numbers in the atomic_data data structure. It's ok to have
+      // multiple symbols map to the same atomic number (above), but we need to
+      // be sure that we only store one entry per atomic number.
```
Comment (bp-kelley) [inline Code/GraphMol/PeriodicTable.cpp]: I'm in agreement with @ptosco, I would add a small comment like // n.b. AtomicNum 0 is a dummy atom :)

**Comment by d-b-w** [comment]
I was thinking about how we could have avoided the bug that lead to this case if there are future edits to the `periodicTableAtomData` string. I don't think we need to add any new features, I just wanted to raise the issue of the bug at this point, when we remember that it happened.

I'm fine resolving/dropping/ignoring my concern if we feel that the names in the periodic table are now more stable than they were when the `periodicTableAtomData` string was first added to RDKit.

**Comment by greglandrum** [comment]
I added a comment in that affected bit of `atomic_data.cpp` trying to clarify the situation. @d-b-w : does that address your concerns?

(I'm reluctant to add the specific check you're looking for in case we end up with three-character termporary atomic symbols again)

**Comment by d-b-w** [comment]
Absolutely. Thanks for humoring me :)

File: Code/GraphMol/PeriodicTable.cpp (hunk 1 above)
Comment (greglandrum) [reply Code/GraphMol/PeriodicTable.cpp]: that's done

**Pull request merged**